frontend/src/store/NotificationState.ts

```diff
@@ -35,7 +35,6 @@ export const setNotificationMsg = (
     msg: string = '',
     fullMsg: string = ''
 ) => {
-    resetNotification()
```

Comment:
This change does work. However, some issues:
* The error message is not clear:
![image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/55c948da-6862-4aaa-a61f-c37421fec7c0)
My understanding is this is the Case 1.

* ManagePermissions appears to have a bug for "delete" action.
Maybe I am not clear. So when the messages displayed already (previous case 1), and user still stays on the same component, and  user deletes a role assignment; then what do we expect to see?
Do we reset all previous messages and display current message (success/fail)?
![image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/9a5f22f3-9e1a-4707-9232-9b4c49b06c99)

Reply:
Hello Ian! The case one will not be entering in this ticket I believe, Catherine and Olga will work on the design first if I'm not mistaken, and about the "delete" action, I think we would need to ask @OlgaLiber2. Sorry for the late reply!